packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -98,14 +98,10 @@ class GoogleMapController {
   /// platform side.
   ///
   /// The returned [Future] completes after listeners have been notified.
-  Future<void> _updateMapOptions(Map<String, dynamic> optionsUpdate) async {
+  Future<void> _updateMapOptions(Map<String, dynamic> optionsUpdate) {
```

Comment:
nits: prefer async/await
ditto below

Reply:
Since these methods are only pass-throughs to the platform interface implementation, there's no need to `await` for anything (or to mark the function as `async`. We just return whatever future comes back from the platform.

The plugin needs to `await` the call to `controller._updateMapOptions()` anyway, so there's not a lot of value in having an `await` in the middle (that doesn't do anything with the returned value).